Assets/Scripts/GameProcess/Main.cs

```diff
@@ -31,7 +29,7 @@ private PreparationLogicHolder LoadPlayerSelection()
         PreparationLogicHolder holder = Instantiate(preparationLogicHolderPrefab, null);
         holder.name = "PreparationLogicHolder";
 
-        PreparationLogic logic = PreparationLogic.LoadFromFile(System.IO.Path.Combine(gameObject.scene.path, "../playerSelection.xml"));
+        PreparationLogic logic = PreparationLogic.LoadFromFile(System.IO.Path.Combine(gameObject.scene.path, "../debug/playerPreparation.xml"));
```

Comment:
Why have you changed it? Currently card selection is saved to `System.IO.Path.Combine(gameObject.scene.path, "../playerSelection.xml")` (see [here]( https://github.com/OriginalUtkin/board_game/blob/ea4ab0ef53066dcab34932475d91b817400b7316/Assets/Scripts/GamePreparation/PreparationMain.cs#L140)). Maybe this is a good sign it can be refactored and placed to static method, or something like that (for example, `PreparationLogic.buildPrepFilePath(gameObject.scene.path)`)

Reply:
Probably I did some experiments and forgot to remove it from there. I'd prefer to discuss how we work with that and will replace it as it was for now